corehq/apps/app_manager/xform.py

```diff
@@ -1360,14 +1358,6 @@ def add_meta_2(self, form):
             value="instance('commcaresession')/session/context/appversion"
         )
 
-        if add_meta_drift:
-            self.add_setvalue(
-                ref="meta/drift",
-                event="xforms-revalidate",
-                value="if(count(instance('commcaresession')/session/context/drift) = 1, "
-                      "instance('commcaresession')/session/context/drift, '')",
-            )
-
```

Comment:
I'm confused - isn't this removing the drift expression entirely?  This removes all of the drift related code from both of those PRs.

Reply:
I apologize. Did all that fairly quickly and incorrectly. I meant to just remove the if condition. Added back meta.

Reply:
Ah cool, this makes more sense.